packages/grafana-llm-app/src/README.md

```diff
@@ -24,6 +24,8 @@ Grafana Cloud: the LLM app plugin is installed for everyone, but LLM features ar
 
 OSS or Enterprise: install and configure this plugin with your OpenAI-compatible API key to enable LLM-powered features across Grafana.
 
+For more detailed setup instructions see [LLM plugin](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/llm/llm-setup/) section under machine learning.
```

Comment:
```suggestion
For more detailed setup instructions see [LLM plugin](/docs/grafana-cloud/alerting-and-irm/machine-learning/llm/llm-setup/) section under machine learning.
```

Reply:
one tweak, otherwise lgtm!

Reply:
This linked to the cloud machine-learning docs and not docs in this repo, I think we need the grafana url header for this.